webapp/src/main/java/com/box/l10n/mojito/entity/ThirdPartyFileChecksum.java

```diff
@@ -0,0 +1,82 @@
+package com.box.l10n.mojito.entity;
+
+import javax.persistence.Column;
+import javax.persistence.Embedded;
+import javax.persistence.Entity;
+import javax.persistence.Index;
+import javax.persistence.PreUpdate;
+import javax.persistence.Table;
+import org.hibernate.annotations.Type;
+import org.joda.time.DateTime;
+
+/** Entity that stores the checksum of a translated file downloaded via a third party sync. */
+@Entity
+@Table(
+    name = "third_party_sync_file_checksum",
+    indexes = {
+      @Index(
+          name = "I__THIRD_PARTY__CHECKSUM__REPOSITORY_ID__FILE_NAME__LOCALE_ID",
+          columnList = "repository_id, locale_id, file_name",
+          unique = true),
+    })
+public class ThirdPartyFileChecksum extends SettableAuditableEntity {
+
+  @Embedded private ThirdPartyFileChecksumCompositeId thirdPartyFileChecksumCompositeId;
+
+  @Column(name = "md5")
+  private String md5;
+
+  @Column(name = "last_modified_date")
+  @Type(type = "org.jadira.usertype.dateandtime.joda.PersistentDateTime")
+  private DateTime lastModifiedDate;
```

Comment:
Just realized it's AuditableEntity and not SettableAuditableEntity that has both DateTime fields defined out of the box - would save needing this definition here.